src/pages/ClusterPage/ClusterPage.js

```diff
@@ -170,6 +173,9 @@ export default class ClusterPage extends Component {
 				|| mark.plan.endsWith(cluster.pricing_plan)
 				|| mark.plan.startsWith(cluster.pricing_plan),
 		);
+		console.log('trial: ', isUsingClusterTrial);
+		console.log('clusters: ', cluster);
```

Comment:
We need to remove this logs @ShahAnuj2610

Reply:
these logs were put by @siddharthlatest. He would best know it.